src/main/java/com/epam/ta/reportportal/core/analyzer/impl/AnalyzerServiceImpl.java

```diff
@@ -184,12 +182,15 @@ private void analyzeLaunch(Launch launch, List<TestItem> toAnalyze, IndexLaunch
 				LOGGER.info("Analysis has found a match: {}", analyzed);
 
 				TestItemActivityResource before = TO_ACTIVITY_RESOURCE.apply(testItem, projectId);
-				updateTestItemIssue(projectId, analyzed, testItem);
+				Map<String, String> relevantItemInfo = updateTestItemIssue(projectId, analyzed, testItem);
```

Comment:
let's create and object RelevantItemInfo for such purpose

Reply:
fixed